pycobertura/reporters.py

```diff
@@ -174,6 +174,28 @@ def generate(self):
         )
 
 
+class PhorgeJsonReporter(Reporter):
+    """
+    A reporter that outputs a JSON object in the Phorge/Phabricator
+    coverage format. It is required to have the source files available.
+    """
+
+    def generate_one(self, filename):
+        statuses = self.cobertura.line_statuses(filename)
+        with self.cobertura.filesystem.open(filename) as f:
+            lines = ["N" for _ in f]
```

Comment:
Could we do something like `["N"] * len(f)`? On my phone now so I can't test.

Reply:
I don't think so, as the file handle is not an array at all.

```
] python -c 'with open("setup.cfg") as f: print(len(f))'
Traceback (most recent call last):
  File "<string>", line 1, in <module>
TypeError: object of type '_io.TextIOWrapper' has no len()

] python -c 'with open("setup.cfg") as f: print(len(["N" for _ in f]))'
50
```